src/cluster.c

```diff
@@ -1413,16 +1415,16 @@ void clusterCommandSlots(client *c) {
      *               3) node ID
      *           ... continued until done
      */
-    connTypeForCaching conn_type = connIsTLS(c->conn);
+    connTypeForCaching conn_type = shouldReturnTlsInfo();
```

Comment:
Unrelated to this PR but there is a type mismatch here. both `connIsTLS` and `shouldReturnTlsInfo` effectively return a `bool`. This code works because of the coincidence that the enum value for `CACHE_CONN_TLS` happens to be 1 too.  I think we should fix it as part of this PR.

Also, I think we should agree on the spelling of acronyms. I see both `TLS` and `Tls` in our code. I would suggest for all camel-casing identifiers we capitalize the first letter only, i.e, Tcp/Tls/etc. Thoughts?

Reply:
> Also, I think we should agree on the spelling of acronyms. I see both TLS and Tls in our code. I would suggest for all camel-casing identifiers we capitalize the first letter only, i.e, Tcp/Tls/etc. Thoughts?
 
 Since TLS is an acronym, I think we should always capitalize it.

Reply:
>Since TLS is an acronym, I think we should always capitalize it.

I think the normal is `Tcp`, `Resp`, `Tls`, etc.

Reply:
I can see an almost equal amount of each style, maybe slightly more of the style with capitalized acronyms. Examples:

* streamID, updateLFU, ACLGetCommandID, dbAddRDBLoad, serverGitSHA1, isValidAOFfilename, lookupClientByID
* commandId, anetEnableTcpNoDelay, getTempIncrAofName, getShardIdPingExtSize, VM_GetDbIdFromDefragCtx

Let's accept both styles? :)